lib/grammers-client/src/client/auth.rs

```diff
@@ -180,6 +181,24 @@ impl Client {
         }
     }
 
+    /// Returns data for copying authorization to another data-center.
+    ///
+    /// See https://core.telegram.org/method/auth.exportAuthorization
+    pub async fn export_authorization(
```

Comment:
Initially it was ```pub(crate)``` , I made it ```pub``` because it is part of the telegram API schema (https://core.telegram.org/method/auth.exportAuthorization).
Since the import is non trivial and depends on lots of internal private parts, you're right, I see no reason to expose this.
I will hide this inside ```net.rs```